package.json

```diff
@@ -4,6 +4,7 @@
   "author": "Christopher Jeffrey",
   "version": "0.7.0",
   "main": "./lib/marked.js",
+  "module": "./src/marked.js",
```

Comment:
You should probably use "type: module" to make this work with Node.js (however it is still experimental)

https://nodejs.org/api/esm.html#esm_code_package_json_code_code_type_code_field

Reply:
@UziTech Why did you mark this as "resolved"? This does not follow the Node.js standard for ESM.

Reply:
I just realized this PR is not using ESM at all?

Reply:
Ya ES Modules are still experimental and I found that CommonJS is faster in node lts right now.

It wouldn't be hard to change to ESM in the future.

Reply:
@UziTech  I guess I don't see the difference between this PR and #1349 or #1452

#1349 implements ESM source files and separates into separate modules instead of one source file

#1452 implements CJS => ESM as an output

We rejected both of them with the plan to wait for ESM to go stable in Node. So why make this PR?

Reply:
This PR doesn't move to ESM it uses CommonJS for node. Other than that they are pretty much the same.

When I started creating this PR it was more or less just to test the speed of making marked modular and it turns out moving to CommonJS on node made it a bit faster , and using rollup and babel to output browser versions kept the speed about the same.